webrtc/RTCPeerConnection-getDefaultIceServers.html

```diff
@@ -35,7 +35,7 @@
    */
 
   test(() => {
-    assert_own_property(RTCPeerConnection, 'getDefaultIceServers');
+    assert_idl_attribute(RTCPeerConnection, 'getDefaultIceServers');
```

Comment:
This doesn't look correct.  getDefaultIceServers should be an own property of the interface object, not an inherited property.  I'd just remove this line entirely, because this is already tested in interfaces.html.